js/contest.js

```diff
@@ -6,6 +6,13 @@ function escapeHtml(unsafe) {
       .replace(/"/g, "&quot;")
       .replace(/'/g, "&#039;");
 }
+function getDuration($obj){
+        var hms=$obj.val().replace(/[,;.-\/]/g,":").replace(/[^0-9:]/g,"").split(":");
```

Comment:
I suspect that there will be a datetime related function, if not, find a plugin to do that :)